wg-macros.toml

```diff
@@ -1,26 +1,39 @@
 name = "Rust Macros Working Group"
 description = "Meetings for the macros working group"
 
+[[timezones]]
+name = "Europe/Rome"
```

Comment:
Also, we have been adding the timezones in a common `_timezones.toml` file so that they are shared between every calendar that wants to use them.

Reply:
Ok removed all the timezones entry :)